core/pom.xml

```diff
@@ -270,6 +270,30 @@
                                 </aggregation>
                             </aggregations>
                         </resourcesSet>
+                        <resourcesSet>
+                            <includes>
+                                <include>inputpin/inputpin-widget.js</include>
+                            </includes>
+                            <aggregations>
+                                <aggregation>
+                                    <inputDir>${resources.dir.compressed}</inputDir>
+                                    <outputFile>${resources.dir.compressed}/inputpin/inputpin.js</outputFile>
+                                    <withoutCompress>${withoutCompress}</withoutCompress>
+                                </aggregation>
+                            </aggregations>
+                        </resourcesSet>
+                        <resourcesSet>
+                            <includes>
+                                <include>inputpin/inputpin.css</include>
+                            </includes>
+                            <aggregations>
+                                <aggregation>
+                                    <inputDir>${resources.dir.compressed}</inputDir>
+                                    <outputFile>${resources.dir.compressed}/inputpin/inputpin.css</outputFile>
+                                    <withoutCompress>${withoutCompress}</withoutCompress>
+                                </aggregation>
```

Comment:
You can change this section to be in the above `<include>inputpin/**</include>` and it will compress the JS and CSS and name them `inputpin.js` and `inputpin.css`.